content/en/sensitive_data_scanner/_index.md

```diff
@@ -28,67 +28,121 @@ Sensitive data, such as credit card numbers, bank routing numbers, and API keys
+3. Install the Datadog Agent v7.54 or newer.
+
+[1]: /account_management/rbac/permissions/#compliance
+[2]: /account_management/rbac/
+[3]: /agent/remote_config/?tab=configurationyamlfile#enabling-remote-configuration
+{{% /tab %}}
+{{< /tabs >}}
 
 ### Add a scanning group
 
-A scanning group determines what data to scan. It consists of a query filter and a set of toggles to enable scanning for logs, APM, RUM, and events. See the [Log Search Syntax][4] documentation to learn more about query filters.
+{{< tabs >}}
+{{% tab "In the Cloud" %}}
+A scanning group determines what data to scan. It consists of a query filter and a set of toggles to enable scanning for logs, APM, RUM, and events. See the [Log Search Syntax][2] documentation to learn more about query filters.
 
-For Terraform, see the [Datadog Sensitive Data Scanner group][5] resource.
+For Terraform, see the [Datadog Sensitive Data Scanner group][3] resource.
 
-To set up a scanning group:
+To set up a scanning group, perform the following steps:
 
-1. Navigate to the [Sensitive Data Scanner][6] configuration page. Alternatively, go to **Organization Settings** > **Manage Sensitive Data** and click the **Configuration** tab.
+1. Navigate to the [Sensitive Data Scanner][1] configuration page.
 1. Click **Add scanning group**. Alternatively, click the **Add** dropdown menu on the top right corner of the page and select **Add Scanning Group**.
 1. Enter a query filter for the data you want to scan. At the top, click **APM Spans** to preview the filtered spans. Click **Logs** to see the filtered logs.
 1. Enter a name and description for the group.
 1. Click the toggle buttons to enable Sensitive Data Scanner for the products you want (for example, logs, APM spans, RUM events, and Datadog events).
 1. Click **Create**.
 
+[1]: https://app.datadoghq.com/organization-settings/sensitive-data-scanner/configuration
+[2]: /logs/explorer/search_syntax/
+[3]: https://registry.terraform.io/providers/DataDog/datadog/latest/docs/resources/sensitive_data_scanner_group
+{{% /tab %}}
+{{% tab "Using the Agent" %}}
+<div class="alert alert-warning"><strong>Note</strong>: Sensitive Data Scanner using the Agent supports only one scanning group per organization.</div>
+
+A scanning group determines what logs to scan. It consists of a query filter to match eligible agents based on host tags.
+
+To set up a scanning group, perform the following steps:
+
+1. Navigate to the [Sensitive Data Scanner using the Agent][1] configuration page.
+1. Click **Add scanning group**. Alternatively, click the **Add** dropdown menu on the top right corner of the page and select **Add Scanning Group**.
+1. Enter a query filter for the data you want to scan. You can use only host-level tags for matching agents. At the bottom, the number of matching and eligible agents is displayed, including the total number out of all agents that match the tag.
+1. Enter a name and description for the group.
+1. Click **Save**.
```

Comment:
It should be the same name as for cloud. I will see with the frontend if we can change the keyword here and have the same for Cloud and On-prem
```suggestion
1. Click **Create**.
```

Reply:
I don't want to change this until the actual app is changed.

Reply:
sounds good to me, I open another PR once it's changed on the frontend  (most probably next week)

Reply:
Sounds good